jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -37,7 +37,11 @@
 import java.util.Properties;
 /*[IF Sidecar19-SE-B165]*/
 import jdk.internal.vm.VMSupport;
-/*[ENDIF]*/
+/*[ELSE]*/
```

Comment:
You could add the Sidecar19-SE-B165 to the comment since this is done elsewhere.

Reply:
Done